### PR TITLE
update chlog after 1.8.2

### PR DESCRIPTION
**Pull request by Borda**
## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

File: src/pytorch_lightning/CHANGELOG.md [hunk 1]
```diff
@@ -62,17 +62,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
-### Added
+### Fixed
 
-- Added back the accidentally removed `pytorch_lightning.utilities.distributed.rank_zero_only` function ([#15536](https://github.com/Lightning-AI/lightning/pull/15536))
+- Make sure save_dir can be empty str ([#15638](https://github.com/PyTorchLightning/pytorch-lightning/issues/15638))
+- Fixed the automatic fallback from `Trainer(strategy="ddp_spawn", ...)` to `Trainer(strategy="ddp", ...)` when on an LSF cluster ([#15103](https://github.com/PyTorchLightning/pytorch-lightning/issues/15103))
 
-### Deprecated
 
-- Deprecated `pytorch_lightning.utilities.distributed.rank_zero_only` in favor of `pytorch_lightning.utilities.rank_zero_only` ([#15536](https://github.com/Lightning-AI/lightning/pull/15536))
```
Comment (carmocca) [inline src/pytorch_lightning/CHANGELOG.md:75]: Why are these removed? They were part of the 1.8.1 release

Comment (Borda) [reply src/pytorch_lightning/CHANGELOG.md:75]: probably collision artifacts :)

File: src/pytorch_lightning/CHANGELOG.md [hunk 2]
```diff
@@ -62,17 +62,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+## [1.8.1] - 2022-11-10
```
Comment (Borda) [inline src/pytorch_lightning/CHANGELOG.md:74]:
```suggestion
## [1.8.1] - 2022-11-10
### Deprecated

- Deprecated `pytorch_lightning.utilities.distributed.rank_zero_only` in favor of `pytorch_lightning.utilities.rank_zero_only` ([#15536](https://github.com/Lightning-AI/lightning/pull/15536))
```

**Review by awaelchli — APPROVED** [review]

File: src/pytorch_lightning/CHANGELOG.md [hunk 3]
```diff
@@ -67,13 +67,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 
--
 
-## [1.8.1] - 2022-11-10
+## [1.8.2] - 2022-11-17
 
-### Added
+### Fixed
 
-- Added back the accidentally removed `pytorch_lightning.utilities.distributed.rank_zero_only` function ([#15536](https://github.com/Lightning-AI/lightning/pull/15536))
```
Comment (carmocca) [inline src/pytorch_lightning/CHANGELOG.md:76]: This should be in 1.8.1

File: src/pytorch_lightning/CHANGELOG.md [hunk 4]
```diff
@@ -67,7 +67,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed the automatic fallback from `Trainer(strategy="ddp_spawn", ...)` to `Trainer(strategy="ddp", ...)` when on an LSF cluster ([#15103](https://github.com/PyTorchLightning/pytorch-lightning/issues/15103))
```
Comment (carmocca) [inline src/pytorch_lightning/CHANGELOG.md:67]: Remove this one?

File: src/lightning_lite/CHANGELOG.md [hunk 5]
```diff
@@ -41,11 +41,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 
 
-### Fixed
 
+## [1.8.2] - 2022-11-17
 
-- Fixed the automatic fallback from `LightningLite(strategy="ddp_spawn", ...)` to `LightningLite(strategy="ddp", ...)` when on an LSF cluster ([#15103](https://github.com/PyTorchLightning/pytorch-lightning/issues/15103))
+### Fixed
 
+- Fixed the automatic fallback from `LightningLite(strategy="ddp_spawn", ...)` to `LightningLite(strategy="ddp", ...)` when on an LSF cluster ([#15103](https://github.com/PyTorchLightning/pytorch-lightning/issues/15103))
```
Comment (carmocca) [inline src/lightning_lite/CHANGELOG.md:49]: I don't think we should have a lite changelog yet

**Review by carmocca — APPROVED** [review]

File: src/lightning_lite/CHANGELOG.md (hunk 5 above)
Comment (Borda) [reply src/lightning_lite/CHANGELOG.md:49]: well it is strange to release with no changes :D

Comment (Borda) [reply src/lightning_lite/CHANGELOG.md:49]: anyway, this shall be updated on release PR not here :otter: 
cc: @lantiga

File: src/pytorch_lightning/CHANGELOG.md (hunk 4 above)
Comment (Borda) [reply src/pytorch_lightning/CHANGELOG.md:67]:
```suggestion
```
